pypi_count/command_line_interface.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python -W ignore::DeprecationWarning
+
+"""The command line interface for the PyCount program."""
+from pathlib import Path
+from rich.console import Console
+
+import typer
+
+from pypi_count.py_counter import PyPiCount
+
+cli = typer.Typer()
+
+console = Console()
+
+
+@cli.command()
+def main(  # pylint: disable=R0913,R0912,R0914
+    input_file: Path,
+    class_def: bool = typer.Option(False, "--class-definitions"),
+    import_statements: bool = typer.Option(False, "--import-statements"),
+    comment: bool = typer.Option(False, "--comments"),
+    function_def: bool = typer.Option(False, "--function-definitions"),
+    if_statements: bool = typer.Option(False, "--if-statements"),
+    function_without_docstrings: bool = typer.Option(False, "--function-without-docstrings"),
+    function_with_docstrings: bool = typer.Option(False, "--function-with-docstrings"),
+    class_with_docstrings: bool = typer.Option(False, "--class-with-docstrings"),
+    class_without_docstrings: bool = typer.Option(False, "--class-without-docstrings"),
+    function_parameters: str = typer.Option(None),
+    assignment_statements: bool = typer.Option(False, "--assignment-statements"),
+    augmented_assignment_statements: bool = typer.Option(False,"--augmented_assignment_statements"),
```

Comment:
This option name should be hyphenated to be consistent with all of the other options.